build/android/jni/Android.mk

```diff
@@ -261,7 +261,7 @@ LOCAL_SRC_FILES :=                                \
 # intentionally kept out (we already build openssl itself): jni/src/util/sha256.c
 
 # Network
-LOCAL_SRC_FILES +=                                \
+LOCAL_SRC_FILES += \
```

Comment:
This doesn't align properly and doesn't actually make it look any better anyways.